cmd/messages.go

```diff
@@ -120,17 +120,21 @@ func callLsMessages(opts map[string]string) error {
 	if err != nil {
 		return err
 	}
-	if len(list) < limit {
+	if len(list) <= limit {
```

Comment:
If this is `<=`, you won't ever get anything past the first page. We could add some kind of "more" flag to the response to indicate that there are more results waiting on the node, but that would require some extra database work... may as well just make one more call and get an empty slice in the case where the total results equal limit.